AlphaWallet/Tokens/ViewControllers/TokensCardViewController.swift

```diff
@@ -33,7 +33,7 @@ class TokensCardViewController: UIViewController, TokenVerifiableStatusViewContr
     private let header = TokenCardsViewControllerHeader()
```

Comment:
- [ ] `ButtonsBar` is distorted in `TokensCardViewController` when we tap and hold on a row to show the checkboxes:

<img src="https://user-images.githubusercontent.com/56189/82978805-01ea1a00-a018-11ea-922f-0bb6bcbc46d5.png" width=200>